tests/ftests/049-sudo-systemd_create_scope.py

```diff
@@ -63,7 +63,7 @@ def teardown(config, result):
     pid = CgroupCli.get(config, cgname=os.path.join(SLICE, SCOPE), setting='cgroup.procs',
                         print_headers=False, values_only=True)
     if pid is not None:
-        Run.run(['kill', '-9', str(pid)], shell_bool=True)
```

Comment:
The `str()` cast isn't needed in this case because the PID came from `cgget` which returns it as a string already